bin/launcher

```diff
@@ -7,11 +7,11 @@ if [ -z ${RETRO_CORE} ]; then
 	RETRO_CORE=${2}
 fi
 
-SAVE_DIR=${DATA_HOME}/steam-buddy/saves/${1}
-RETRO_CONFIG=/tmp/steam-buddy-retroarch.cfg
+SAVE_DIR=${DATA_HOME}/chimera/saves/${1}
+RETRO_CONFIG=/tmp/chimera-retroarch.cfg
 
-CORE_CONFIG=/usr/share/steam-buddy/config/${1}.cfg
-LOCAL_CONFIG=${CONFIG_HOME}/steam-buddy/${1}.cfg
```

Comment:
We will need to do something to migrate all the various directories.

Reply:
Yeah. We could add a simple script to move `.local/share/steam-buddy/` to `.local/share/chimera/` and all other related directories.

Can be run by `steam-tweaks` itself as a first step.